vendor/inline-scripts/tracking-nielsen.js

```diff
@@ -1,56 +0,0 @@
-(function (M) {
```

Comment:
are you sure we want to remove it completely?

Reply:
Yep, it is handled in ads bundle.